idaes/models_extra/column_models/tests/test_MEAsolvent_column.py

```diff
@@ -55,21 +54,21 @@ def model(self):
         m.fs = FlowsheetBlock(default={"dynamic": False})
     
         # Set up molar flow bounds
-        vaporconfig_absorber["state_bounds"]["flow_mol"] = (0, 1, 1e6, pyunits.mol / pyunits.s)
-        liquidconfig["state_bounds"]["flow_mol"] = (0, 1, 1e6, pyunits.mol / pyunits.s)
+        vaporconfig_copy = vaporconfig.copy()
```

Comment:
I think you need to do `deepcopy` here - hopefully that will fix the issues with the solvent condenser test.